src/styles/global.css

```
@@ -1,116 +1,116 @@
+    text-align: left;
+    background-color: #1a1b26;
+    background-size: 100% 600px;
+    word-wrap: break-word;
+    overflow-wrap: break-word;
+    color: var(--color-text-primary, var(--text-light));
+    font-size: 20px;
+    line-height: 1.7;
 }
 
 /* Prevent white flash during view transitions */
 ::view-transition-old(root),
```

Biome (2.1.2)

[error] Unknown type selector is not allowed.

See MDN web docs for more details.
Consider replacing the unknown type selector with valid one.

(lint/correctness/noUnknownTypeSelector)